lib/pg_query/deparse.rb

```diff
@@ -1068,6 +1070,14 @@ def deparse_drop(node)
       output.join(' ')
     end
 
+    def deparse_explain(node)
+      output = ['EXPLAIN']
+      options = node.fetch('options', []).map { |option| option['DefElem']['defname'].upcase }
+      output.concat(options)
```

Comment:
We should probably use the `EXPLAIN (ANALYZE, BUFFERS)` syntax here, so multiple elements work correctly

Reply:
I didn't even know that was possible ;)